### PR TITLE
Support CS API v0.6.0

### PR DESCRIPTION
**Pull request by anoadragon453**
Figured I'd get the ball rolling on this one so we don't forget about it.

Fixes https://github.com/matrix-org/synapse/issues/6045

### CODE REVIEW

**Comment by babolivier** [comment]
Just to be sure, are we sure we're supporting all of the features of this version?

**Comment by erikjohnston** [comment]
Is this something that still needs to be done?

**Comment by richvdh** [comment]
er yes

**Comment by anoadragon453** [comment]
We need to briefly check that client folk are correctly using r0.6.0 as a substitute for the removed unstable feature flags.

**Comment by clokep** [comment]
Does it matter that there's now a 0.6.1?

**Comment by anoadragon453** [comment]
> Does it matter that there's now a 0.6.1?

Yes, though we may want to do it in a separate PR. I believe `org.matrix.msc2432` can be removed for 0.6.1.

File: synapse/rest/client/versions.py [hunk 1]
```diff
@@ -49,24 +49,11 @@ def on_GET(self, request):
...
-                    #
-                    # This is also used by a client when they wish to bind a 3PID to their
-                    # account, but not bind it to an identity server, the endpoint for which
-                    # also requires `id_server`. If the homeserver is handling 3PID
-                    # verification itself, there is no need to ask the user for `id_server` to
-                    # be supplied.
-                    "m.require_identity_server": False,
-                    # as per MSC2290
-                    "m.separate_add_and_bind": True,
+                    "m.lazy_load_members": True,
```
Comment (richvdh) [inline synapse/rest/client/versions.py:56]: this is something new? can it have a comment with an MSC link or so?

Comment (anoadragon453) [reply synapse/rest/client/versions.py:56]: Ah, good catch. Looks like this was removed in https://github.com/matrix-org/synapse/commit/dc3f9987061e24bec77ad9e26a7679a1907cee4f#diff-e591d42d30690ffb79f63bb726200891 and was accidentally kept while rebasing this branch.

**Pull request merged**